src/vs/editor/contrib/inlineCompletions/browser/inlineCompletionsController.ts

```diff
@@ -148,6 +154,14 @@ export class InlineCompletionsController extends Disposable {
 			this.updateObservables(tx, getReason(e))
 		)));
 
+		this._register(this._contextKeyService.onDidChangeContext((e) => transaction(tx => {
+			if (e.affectsSome(new Set('editorDictation.inProgress'))) {
+				/** @description contextKeyService.editorDictation.inProgress*/
+				this.updateObservables(tx, VersionIdChangeReason.Other);
+				this.model.get()?.stop(tx);
+			}
+		})));
```

Comment:
I don't think you need this either. When `_enabled` changes, everything that depends on it updates.